template-parts/header_content-archive.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Header content template for the 'Section' page template
+ */
+?>
+
+<?php
+global $post;
+
+$title_string = wptexturize( ucfwp_get_header_title( null ) );
+
+ob_start();
+
+?>
+<h1 class="self-align-middle text-uppercase h4 mb-0"><?php echo $title_string; ?></h1>
```

Comment:
`self-align-middle`: did you mean `align-self-middle`?

Reply:
Removed the class. That was leftover from me trying to figure out how to place layout things properly.